app/utils/shared/UrlUtils.ts

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
-import Const from '../consts';
 import { getUrlWithQueryString } from '../../../backend/sharedUtils';
```

Comment:
Did we omitted this line or is troublesome to also modify this path? 🤔 
```suggestion
import { getUrlWithQueryString } from 'backend/sharedUtils';
```